scaffold/github/actions/pantheon/clone-env/action.yml

```diff
@@ -24,7 +24,7 @@ runs:
           source .github/actions/drainpipe/set-env/bash_aliases
           drainpipe_exec "terminus auth:login --machine-token=\"${{ inputs.terminus-token }}\""
           if [ "${{ inputs.source-site-name }}" == "${{ inputs.target-site-name }}" ]; then
-            drainpipe_exec "terminus env:clone-content ${{ inputs.source-site-name }}.${{ inputs.source-environment }} ${{ inputs.target-environment }}"
+            drainpipe_exec "terminus -y env:clone-content ${{ inputs.source-site-name }}.${{ inputs.source-environment }} ${{ inputs.target-environment }}"
```

Comment:
The long --yes could be better for readability. https://architecture.lullabot.com/adr/20211006-avoid-command-aliases/ Not a blocker to merging.

Reply:
That's what I had originally and switched it to this to match the way this was done on line 35.
https://github.com/Lullabot/drainpipe/pull/103/files#diff-f5cbaf15e8f982c405c560523081b8ba10345b91b59598e57ccf817a65178860R35